language/move-analyzer/editors/code/yarn.lock

```diff
@@ -0,0 +1,2332 @@
+# THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.
```

Comment:
This file is autogenerated - should it really be part of this PR?

Reply:
Yes, you can refer to this article: https://rotadev.com/should-i-commit-the-yarn-lock-file-and-what-is-it-for-dev/

Reply:
I guess what I am asking is - we did not have (nor presumably needed) this file in the past and I am not what makes it required now.

Reply:
Got it, I added it to the ignore file